### DISCUSSION TITLE
coffee machine

### DISCUSSION THREAD

**Discussion by jomagese** [thoukydides/homebridge-homeconnect#120] [Ideas]
Hello, would it be possible to add hot water to the coffee machine? Thank you

**Comment by thoukydides** [comment]
According to the Home Connect API documentation, there are no programs defined for hot water:
https://api-docs.home-connect.com/programs-and-options?#coffee-machine

You will need to request the addition of a new program:
https://developer.home-connect.com/support/contact

This plugin automatically supports all programs that the appliance advertises via the API. If a new program gets added for hot water then it will be supported without requiring any changes to this plugin.
